ext/json/json.c

```diff
@@ -179,21 +179,20 @@ PHP_JSON_API zend_result php_json_decode_ex(zval *return_value, const char *str,
 /* }}} */
 
 /* {{{ */
-PHP_JSON_API zend_result php_json_validate_ex(zval *return_value, const char *str, size_t str_len, zend_long options, zend_long depth)
+PHP_JSON_API zend_result php_json_validate_ex(const char *str, size_t str_len, zend_long options, zend_long depth)
```

Comment:
```suggestion
PHP_JSON_API bool php_json_validate_ex(const char *str, size_t str_len, zend_long options, zend_long depth)
```

In addition to changing the return values to `bool` true/false, the return type should also be changed to `bool`.

zend_result is defined as the SUCCESS/FAILURE macro

```c
typedef enum {
  SUCCESS =  0,
  FAILURE = -1,		/* this MUST stay a negative number, or it may affect functions! */
} ZEND_RESULT_CODE;

typedef ZEND_RESULT_CODE zend_result;
```

Reply:
:heavy_check_mark: